save-orchestrator/src/test/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 orchestrator.backendUrl=http://backend:5000
+orchestrator.agentLogsRelativePath=cnb/agentLogs
 orchestrator.testResources.basePath=/home/cnb/repositories
```

Comment:
Why do we have an absolute path for repositories, but can't for agent logs?

Reply:
Because we override this path. I fixed